Profiler/CProfilerCallback.cpp

```diff
@@ -3,7 +3,6 @@
 #include "UploadDaemon.h"
 #include "utils/StringUtils.h"
 #include "utils/WindowsUtils.h"
-#include "utils/Debug.h"

```

Comment:
I'd prefer not to make this part of the classes interface. See also my comment at the end about the singleton pattern